config/config.go

```diff
@@ -134,10 +134,12 @@ type GrafanaConfig struct {
 	Password                 string `yaml:"password"`
 }
 
-// JaegerConfig describes configuration used for jaeger links
-type JaegerConfig struct {
-	URL     string `yaml:"url"`
-	Service string `yaml:"service"`
+// TracingConfig describes configuration used for tracing links
+type TracingConfig struct {
+	EnableJaeger bool
```

Comment:
This boolean is confusing to me. What is it used for? I do not see it being set in any configuration setting, and its hardcoded default is "false". Perhaps you should add a comment somewhere to explain what this is for. If this is NOT meant to come from the config file, it might also be good to add "yaml:-" here to make it clear it doesn't belong in the configmap yaml and will be skipped:

```
   EnableJaeger bool `yaml:-`
```

Reply:
it's hardcoded but change in the discover, this flag let kiali if he should use jaeger or not. I'll add comments